doc/api/dns.md

```diff
@@ -459,12 +459,12 @@ will be present on the object:
 
 Here is a example of the `ret` object passed to the callback:
 
-<!-- eslint-disable -->
+<!-- eslint-disable semi -->
 ```js
 [ { type: 'A', address: '127.0.0.1', ttl: 299 },
   { type: 'CNAME', value: 'example.com' },
   { type: 'MX', exchange: 'alt4.aspmx.l.example.com', priority: 50 },
-  { type: 'NS', value: 'ns1.example.com', type: 'NS' },
+  { type: 'NS', value: 'ns1.example.com' },
```

Comment:
This fix illustrates why we should use a rule-disabling instead of full disabling when it is possible.